pyoozie/tags.py

```diff
@@ -494,6 +496,133 @@ def _xml(self, doc, tag, text):
         return doc
 
 
+class _WorkflowEntity(typing.Iterable):
```

Comment:
`_AbstractWorkflowEntity` would be more precise (since these are execution control nodes contained within a workflow), but it's also really verbose. What is the convention for this? I can find none in the [Google styleguide](https://google.github.io/styleguide/pyguide.html) or [`shopify_python`](https://github.com/Shopify/shopify_python).